Features/MovieTabBar/MovieTabBarItem.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  MovieTabBarItem.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 3.08.22.
+//
+
+import Foundation
+import UIKit
```

Comment:
If you import UIKit you don't need to import Foundation as well
```suggestion
import UIKit
```